### PR TITLE
More verbose error message on exceptions.

### PR DESCRIPTION
**Pull request by antmat**
subj. Same as https://github.com/3Hren/cocaine-core/pull/34

### CODE REVIEW

**Comment by kobolog** [comment]
Have you seen how it looks like with this change?

File: src/errors.cpp [hunk 1]
```diff
@@ -188,6 +188,11 @@ security_category() -> const std::error_category& {
 
 namespace cocaine { namespace error {
 
+std::string
+what(const std::system_error& e) {
+    return format("[%d] %s", e.code().value(), e.code().message(), e.what());
```
Comment (3Hren) [inline src/errors.cpp]: This won't work (argument count mismatch).

**Comment by 3Hren** [comment]
LGTM, but I'd rather perform rebasing.

**Comment by antmat** [comment]
Closed in favour of #229

**Pull request closed without merging**